virtual_rainforest/core/core_schema.json

```diff
@@ -9,23 +9,44 @@
                "description": "Details of the grid to configure",
                "type": "object",
                "properties": {
-                  "nx": {
+                  "grid_type": {
+                     "description": "The grid cell type",
+                     "type": "string",
+                     "default": "square"
+                  },
+                  "cell_area": {
+                     "description": "The area of each grid cell",
+                     "type": "number",
+                     "exclusiveMinimum": 0,
+                     "default": 100
+                  },
+                  "cell_nx": {
                      "description": "Number of grid cells in x direction",
                      "type": "integer",
                      "exclusiveMinimum": 0,
                      "default": 100
                   },
-                  "ny": {
+                  "cell_ny": {
                      "description": "Number of grid cells in y direction",
                      "type": "integer",
                      "exclusiveMinimum": 0,
                      "default": 100
+                  },
+                  "xoff": {
+                     "description": "The x offset of the grid origin",
+                     "type": "number",
+                     "default": 0
+                  },
+                  "yoff": {
+                     "description": "The y offset of the grid origin",
+                     "type": "number",
+                     "default": 0
                   }
                },
                "default": {},
                "required": [
-                  "nx",
-                  "ny"
+                  "cell_nx",

```

Comment:
Not really an important point, but I've just realised that `cell_nx` and `cell_ny` don't need to be required as they have defaults set. I think we could just have `"required": []` here.

Reply:
I'd thought the same - have updated.